Makefile

```diff
@@ -121,8 +117,18 @@ gendoc:
 		xargs -I% md-to-godoc -input=%
 
 .PHONY: clean
-clean::
+clean:
```

Comment:
All the other targets are single. Double is useful if it's going to be extended by some other file, but these make rules are all self-contained, so semantically single-colon is correct. I think this was added by accident (or copy-pasted from some internal makefiles, which may have had some extensibility concerns we don't have here).